rspec-mode.el

```diff
@@ -567,13 +567,15 @@ file if it exists, or sensible defaults otherwise"
 
 (defun rspec-spring-p ()
   (and rspec-use-spring-when-possible
-       (let ((root (rspec-project-root)))
+       (let ((root (substring (rspec-project-root) 0 -1))
+              (ruby-version (shell-command-to-string "ruby -e 'print RUBY_VERSION'")))
          (or
           ;; Older versions
-          (file-exists-p (concat root "tmp/spring/spring.pid"))
+          (file-exists-p (format "/%s/tmp/spring/spring.pid" root))
           ;; 0.9.2+
-          (file-exists-p (concat temporary-file-directory "spring/"
-                                 (md5 (substring root 0 -1)) ".pid"))))))
+          (file-exists-p (format "/%s/spring/%s.pid" temporary-file-directory (md5 root)))
```

Comment:
Yes, it becomes '//'. This is just to be sure that path is always absolute, same for '/spring/', which could be '/spring' in some cases. I've taken this approach because
1. It will never fail, should arguments contain slashes or not.
2. It is easier because you don't need to remember which arguments do contain slashes and which don't.
Though there are also some network-related protocols which use two leading slashes. I'll better change it.